hathor/verification/vertex_verifier.py

```diff
@@ -44,6 +45,25 @@ class VertexVerifier:
     def __init__(self, *, settings: HathorSettings) -> None:
         self._settings = settings
 
+    def verify_version(self, vertex: BaseTransaction) -> None:
+        """Verify that the vertex version is valid."""
+        valid_versions = [
```

Comment:
You can create global sets with these versions and just use them here. You might even refactor it to `self._settings.CONSENSUS_ALGORITHM.get_valid_versions()`.

Note: Remember to use a `set()` instead of a `list()` so we can check if an element exists in O(1).

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1060/commits/157044e8acb40e2ed85ee1a35b983d1c8df46e3f

Reply:
Great! Thanks!